CMakeLists.txt

```diff
@@ -425,6 +425,10 @@ elseif (ENABLE_STDCXX_SYNC)
 	endif()
 endif()
 
+if (ENABLE_MUTEX_DB)
+	add_definitions(-DENABLE_MUTEX_DB=1)
+endif()
```

Comment:
Maybe better to follow the `SRT_DEBUG_*` path? Like `SRT_DEBUG_TSBPD_DRIFT`.
`SRT_DEBUG_MUTEXDB`